src/main/terrier.cpp

```diff
@@ -42,11 +45,18 @@ int main() {
   // initialize stat registry
   auto main_stat_reg = std::make_shared<terrier::common::StatisticsRegistry>();
 
+  // create the global transaction mgr
+  terrier::storage::RecordBufferSegmentPool buffer_pool_(100000, 10000);
```

Comment:
How these numbers are set? Do we need to dynamically adjust them? Should they hook up with the settings manager in the future?

Reply:
created issue #346